internal/explorer/models.go

```diff
@@ -174,21 +158,17 @@ type nodeWithNestedCapacity struct {
 	Uptime            int64           `json:"uptime"`
 	Created           int64           `json:"created"`
 	FarmingPolicyID   int             `json:"farmingPolicyId"`
-	UpdatedAt         string          `json:"updatedAt"`
+	UpdatedAt         int64           `json:"updatedAt"`
```

Comment:
updatedAt needs to be converted to string for the explrorer to correctly recognize it?